pages/posts/index.tsx

```diff
@@ -60,17 +85,29 @@ const PostList = () => {
         </CommonAppBar.Left>
         <CommonAppBar.Right>
           <CommonIconButton iconName="share" alt="공유" />
-          <CommonIconButton iconName="more" alt="더보기" />
+          <CommonIconButton
+            iconName="more"
+            alt="더보기"
+            onClick={toggleSheet()}
+          />
         </CommonAppBar.Right>
       </CommonAppBar>
       <PostListContainer>
         {postList.map((post) => (
           <li key={post.id}>
-            <PostItem post={post} />
+            <PostItem post={post} isEditing={isEditing} />
```

Comment:
```suggestion
            <PostItem {...{post, isEditing}} />
```
props가 길어지면 이런식으로도 줄이더라구요 공유차원에 알려드립니다 ㅎㅎ.. 전 지금 방식도 좋슴다

Reply:
오 괜찮은 것 같네여... 근데 props 이름이 계속 같다는 보장이 없어서 확장성 측면에서는 아쉬울 수 있을 것 같아옄ㅋㅋ

Reply:
아 근데 그냥 저건 저렇게 넣어주고, 이름 다르면 그냥 넣어주면되니까 문제 없겠네여... 자문자답...

Reply:
저도 좋은 것 같아요~~!

Reply:
[7b61386](https://github.com/depromeet/11th_5team_fe/pull/88/commits/7b6138668e6988f92d5f4dd706354847125944da) 반영했습니다~!